### PR TITLE
Make the AVIF test images render larger

### PR DESCRIPTION
**Pull request by gsnedders**
At the moment, they're 2px x 2px images, which is kinda hard to notice on a large screen. Make this scale much larger, which should work fine.

### CODE REVIEW

File: avif/animated-avif-timeout-ref.html [hunk 1]
```diff
@@ -1 +1 @@
-<img src=../images/green.avif>
+<img src=../images/green.avif style="height:500px">
```
Comment (mathiasbynens) [inline avif/animated-avif-timeout-ref.html:1]: Alternatively, `width=500 height=500`?

File: avif/animated-avif-timeout.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <html class="reftest-wait">
 <title>Animated AVIF: Second frame displays quickly, replacing red with green.</title>
 <link rel="match" href="animated-avif-timeout-ref.html"/>
-<img src=../images/animated-avif.avif onload="loaded()"/>
+<img src=../images/animated-avif.avif onload="loaded()" style="height:500px"/>
```
Comment (mathiasbynens) [inline avif/animated-avif-timeout.html:4]: Same here

**Review by mathiasbynens — APPROVED** [review]
While manually testing these I was previously using browser / page zoom, but this patch would make that unnecessary 👍 

LGTM, with just a question: is there any reason to prefer `style` over `width`/`height` for this purpose?

**Comment by gsnedders** [comment]
> LGTM, with just a question: is there any reason to prefer `style` over `width`/`height` for this purpose?

I no longer know. `width`/`height` does stuff to the intrinsic ratio, does it not?

**Comment by mathiasbynens** [comment]
> > LGTM, with just a question: is there any reason to prefer `style` over `width`/`height` for this purpose?
> 
> I no longer know. `width`/`height` does stuff to the intrinsic ratio, does it not?

Ack. Either way it doesn’t matter for these tests so let’s just land this. Thanks!

**Pull request merged**